query/src/main/scala/filodb/query/exec/MetadataRemoteExec.scala

```diff
@@ -39,17 +44,42 @@ case class MetadataRemoteExec(queryEndpoint: String,
   }
 
   def toQueryResponse(response: MetadataSuccessResponse, id: String, parentSpan: kamon.trace.Span): QueryResponse = {
-    val iteratorMap = response.data.map { r => r.map { v => (v._1.utf8, v._2.utf8) }}
+      if (response.data.isEmpty) mapTypeQueryResponse(response, id)
```

Comment:
Can any DistConcat fail somewhere if we consider empty response as mapTypeResponse ?